ets-demo/etsdemo/app.py

```diff
@@ -873,6 +873,9 @@ def get_children(self):
             editor=HTMLEditor(
                 format_text=True,
                 base_url_name='base_url',
+                # Open links externally, in the default browser, instead of
+                # in the editor.
```

Comment:
I guess the attribute name is _almost_ expressive enough to not require a separate comment. `open_link_externally` would have been a better name, but it is harder to change a published public API. Would an inline `open_externally=True,    # for links` be informative enough without hurting readability?

Reply:
I'd just drop the comment.  `HTMLEditor` is reasonably well documented so it can be looked up if there is a question about what it does.